### ISSUE TITLE
Shell layer broken by changes to xterm-color package

### ISSUE THREAD

**Issue by skomanduri** [syl20bnr/spacemacs#7393]
#### Description :octocat: ####

A [recent commit](https://github.com/atomontage/xterm-color/commit/a97e70fd0ae2cddb51c8c0ca6a8da066649f98f3) to xterm-color removed the function `xterm-color-unfontify-region` which is referenced here:
https://github.com/syl20bnr/spacemacs/blob/c788da709bb1c74344f5ab1b6f18cfdf6b930df8/layers/%2Btools/shell/packages.el#L253

I had to comment out the shell layer to get Spacemacs to start up properly.

#### Reproduction guide :beetle: ####
- Start Emacs
- Add shell layer
- Restart Emacs

*Observed behaviour:* :eyes: :broken_heart:
Startup throws error:
```
font-lock-unfontify-region: Symbol’s function definition is void: xterm-color-unfontify-region
```

The startup buffer doesn't completely load, and color-based syntax highlighting is broken.

*Expected behaviour:* :heart: :smile:
No error is thrown and Spacemacs loads normally

#### System Info :computer: ####
- OS: darwin
- Emacs: 25.1.1
- Spacemacs: 0.200.1
- Spacemacs branch: master (rev. 3706a42)
- Graphic display: t
- Distribution: spacemacs
- Editing style: vim
- Completion: helm
- Layers:
```elisp
(auto-completion chrome clojure emacs-lisp ess eyebrowse git go helm html javascript latex markdown
                 (org :variables org-enable-github-support t)
                 (python :variables python-enable-yapf-format-on-save t python-auto-set-local-pyenv-version 'on-project-switch)
                 (ruby :variables ruby-version-manager 'rbenv)
                 ruby-on-rails shell spell-checking syntax-checking yaml)
```

**Comment by hidaris** [comment]
PR[#7392](https://github.com/syl20bnr/spacemacs/pull/7392)

**Comment by vmfhrmfoaj** [comment]
You can be temporarily resolved as follows:
``` emacs-lisp
;; in .spacemacs
(defun xterm-color-unfontify-region (&rest _))
```

**Comment by sebhahn** [comment]
I can confirm this issue!

**Comment by vmfhrmfoaj** [comment]
You can be temporarily resolved as follows:

``` emacs-lisp
(fset 'xterm-color-unfontify-region 'font-lock-default-unfontify-region)
```

**Comment by sebhahn** [comment]
@vmfhrmfoaj your solutions works! Thanks!

**Comment by TheBB** [comment]
Thanks everyone! Fixed in develop and released as 0.200.2.
